portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,73 @@
+body {
+  font-family: "Sanchez", Verdana, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  background: url("/images/moroccan-flower-dark.png"), #6F4A38;
```

Comment:
Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain: Put declarations in alphabetical order in order to achieve consistent code in a way that is easy to remember and maintain.